src/main/java/lotto/view/OutputView.java

```diff
@@ -28,26 +31,37 @@ private static void outputLottoNumber(List<Integer> lottoNumber) {
     }
 
     public static void outputLottoStatistics(LottoStatisticsDto dto) {
-        Map<Integer, Integer> matchingCounts = dto.getMatchingCounts();
-        Map<Integer, Integer> matchingPrices = dto.getMatchingPrices();
-        System.out.println("\n" + "당첨 통계");
+        Map<LottoRank, Integer> matchingCounts = dto.getMatchingCounts();
+        System.out.printf("%n당첨 통계%n");
         System.out.println("---------");
 
-        for (int key : matchingCounts.keySet()) {
-            System.out.printf("%d" + "개 일치 (" + "%d" + "원)- " + "%d" + "개" + "\n",
-                    key, matchingPrices.get(key), matchingCounts.get(key));
+        List<LottoRank> ranks = new ArrayList<>(matchingCounts.keySet());
+        Collections.sort(ranks, Collections.reverseOrder());
+        for (LottoRank rank : ranks) {
+            outputStatisticsStage(matchingCounts, rank);
         }
 
         outputGrossRateOfEarnings(dto);
     }
 
+    private static void outputStatisticsStage(Map<LottoRank, Integer> matchingCounts,
+                                              LottoRank rank) {
+        if (rank == LottoRank.SECOND) {
```

Comment:
메시지를 보내서 처리해보면 어떨까요?
```suggestion
        if (rank.isSecond()) {
```